### PR TITLE
Migrate glossary terms from Wiki

### PR DESCRIPTION
**Pull request by nadinequinr3**
Preview: https://nadine.preview.docs.r3.com/en/platform/corda/5.0/introduction/glossary.html

### CODE REVIEW

File: content/en/platform/corda/5.0/glossaryitems.json [hunk 1]
```diff
@@ -1,37 +1,262 @@
...
+    "DLT": {
+        "definition": "Distributed Ledger Technology. A type of database technology that enables a network of computers to maintain a shared and synchronized database. It is a decentralized and transparent ledger that records transactions in a tamper-evident and secure manner, making it well-suited for use cases that require high levels of security, trust, and transparency.",
+        "more": ""
+    }, 
+    "ECDH": {
+        "definition": "Elliptic-curve Diffie–Hellman. A key sharing algorithm, most commonly used to send encrypted messages. ",
+        "more": "ECDH works by multiplying your private key by another's public key to get a shared secret, then using that shared secret to perform symmetric encryption."
+    }, 
+    "Entity": {
+        "definition": "An organisation or individual that participates in one ore more application networks that can provide attestation that they are whom they claim to be.",
```
Comment (NeilAndersonR3) [inline content/en/platform/corda/5.0/glossaryitems.json:83]:
```suggestion
        "definition": "An organization or individual that participates in one or more application networks that can provide attestation that they are whom they claim to be.",
```

File: content/en/platform/corda/5.0/glossaryitems.json [hunk 2]
```diff
@@ -1,37 +1,262 @@
...
+    "Bootstrapping": {
+        "definition": "To boot or to load a program using a much smaller initial program to load in the desired program.",
+        "more": "This is usually an operating system or setup files that can be implemented automatically or manually."
+    }, 
+    "CA": {
+        "definition": "Certificate Authority. The holders of a PKI trust root that can issue certificates to customers." ,
+        "more": "When you set up a membership group, you must decide which certificate authorities apply to various identities: TLS, ledger, and REST user identities."
+    }, 
+    "Cluster": {
+        "definition": "A complete set of worker processes. Clusters require a fully-functioning virtual node infrastructure",
```
Comment (NeilAndersonR3) [inline content/en/platform/corda/5.0/glossaryitems.json:43]: Does fully functioning get a hyphen?
```suggestion
        "definition": "A complete set of worker processes. Clusters require a fully functioning virtual node infrastructure".
```

File: content/en/platform/corda/5.0/glossaryitems.json [hunk 3]
```diff
@@ -1,37 +1,262 @@
...
+        "more": "The cluster hosts a dynamically-configured set of virtual nodes, which are equivalent to nodes used in Corda 4."
+    }, 
+    "Corda Host": {
+        "definition": "A complete set of worker processes. Clusters require a fully-functioning virtual node infrastructure",
```
Comment (NeilAndersonR3) [inline content/en/platform/corda/5.0/glossaryitems.json:47]:
```suggestion
        "definition": "A complete set of worker processes. Clusters require a fully functioning virtual node infrastructure".
```

**Review by NeilAndersonR3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0/glossaryitems.json (hunk 3 above)
Comment (nadinequinr3) [reply content/en/platform/corda/5.0/glossaryitems.json]: Missing full stop. Resolved now.

**Review by NeilAndersonR3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @nadinequinr3

**Pull request merged**